packages/framework/src/Component/Redis/RedisFacade.php

```diff
@@ -11,19 +11,38 @@ class RedisFacade
     /**
      * @var \Redis[]
      */
-    protected $cacheClients;
```

Comment:
removed protected property of non final class

Reply:
fixed in d77db04ae